lib/Gateway.js

```diff
@@ -1488,7 +1518,19 @@ Gateway.prototype.discoverValue = function (node, vId) {
         } else {
           cfg.object_id = 'notification_' + valueId.property
         }
-        cfg.discovery_payload.icon = 'mdi:alarm-light'
+        switch (valueId.propertyKey) {
```

Comment:
Add a TODO here to support other notification types, we could do this in another PR if you want (create a feature issue to keep track of this)

Reply:
done this!